post-workshop/post-workshop-ja.md

```diff
@@ -500,13 +500,13 @@ If you have any questions about your rights or treatment as a research participa
 
 -----------
 
-### Thank you screen 
-Thank you for completing this survey! Your input is used to improve our workshops.
+### ありがとうございました
+アンケートにご協力いただきありがとうございました！皆様のご意見は、ワークショップの改善に役立てさせていただきます。
 
-If you believe someone violated The Carpentries [Code of Conduct](https://docs.carpentries.org/topic\_folders/policies/code-of-conduct.html) during the workshop,  we ask that you report it using [this form](https://docs.google.com/forms/d/e/1FAIpQLSdi0wbplgdydl\_6rkVtBIVWbb9YNOHQP\_XaANDClmVNu0zs-w/viewform) (see our [reporting guidelines](https://docs.carpentries.org/topic\_folders/policies/incident-reporting.html) for more information).
+もし、ワークショップ中にCarpentriesの[行動規範](https://docs.carpentries.org/topic\_folders/policies/code-of-conduct.html) に反する行為を見かけた場合は、[このフォーム](https://docs.google.com/forms/d/e/1FAIpQLSdi0wbplgdydl\_6rkVtBIVWbb9YNOHQP\_XaANDClmVNu0zs-w/viewform)で報告してください（詳細は[報告ガイドライン](https://docs.carpentries.org/topic\_folders/policies/incident-reporting.html)をご覧ください）。
```

Comment:
This updates to the Japanese version of the Code of Conduct (as discussed recently and changed elsewhere). Sorry you couldn't have known about this in advance, this saves us a separate conflicting PR. Thanks for the translation 👍 

```suggestion
もし、ワークショップ中にCarpentriesの[行動規範](https://carpentries-coc.readthedocs.io/ja/latest/topic_folders/policies/code-of-conduct.html) に反する行為を見かけた場合は、[このフォーム](https://docs.google.com/forms/d/e/1FAIpQLSdi0wbplgdydl\_6rkVtBIVWbb9YNOHQP\_XaANDClmVNu0zs-w/viewform)で報告してください（詳細は[報告ガイドライン](https://carpentries-coc.readthedocs.io/ja/latest/topic_folders/policies/incident-reporting.html)をご覧ください）。
```